### ISSUE TITLE
Fine-tuning Albert large

### ISSUE THREAD

**Issue by SohaKhazaeli** [google-research/ALBERT#86]
I fine-tuned Albert base on my task but didn't get desired accuracy. Now that I am trying to fine-tune Albert large I get this error:
"Resource exhausted: OOM when allocating tensor with shape[8,512,16,64] and type float on /job:localhost/replica:0/task:0/device:GPU:0 by allocator GPU_0_bfc"

I used single GPU, with 12 GB Memory and also 16 GB (two different attempts).
It is interesting that I can fine-tune Bert base on the single gpu with 12GB memory.

**Comment by 0x0539** [comment]
@dhruvsakalley that was due to a bug in the TF-Hub UI, which should be resolved now. Also, please switch to the "/3" module on TF-Hub.

**Comment by hankcs** [comment]
I managed to finetune albert_large and got better result than albert_base. However, the xlarge yields unreasonable worse result. I doubt if it's due to the hyperparameter or the way I loaded the weights.
